source/features/forked-to.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import cache from '../libs/cache';
+import {getRepoURL} from '../libs/utils';
+import {isOwnRepo} from '../libs/page-detect';
+
+const id = 'forked-to';
+
+const currentRepo = getRepoURL();
+
+// Show cached forks.
+async function showForks(): Promise<void> {
+	const repo = getSourceRepo();
+	const cached = await getCache(repo);
+	const forks = cached.filter(fork => fork !== currentRepo);
+	const pageHeader = select<HTMLElement>('.pagehead h1.public')!;
+	for (const fork of forks) {
+		pageHeader.append(
+			<span className="fork-flag rgh-forked">
+				<span className="text">forked to&nbsp;
+					<a href={`/${fork}`}>{fork}</a>
+				</span>
+			</span>
+		);
+	}
+}
+
+// Store fork if we are on a forked page.
+function checkForForkedPage(): void {
```

Comment:
Instead of adding a comment, name the function appropriately, like
```suggestion
function rememberCurrentRepoIfFork(): void {
```

Or

```suggestion
function rememberCurrentFork(): void {
```